x-pack/plugins/uptime/public/state/api/monitor_management.ts

```diff
@@ -78,3 +78,7 @@ export interface TestNowResponse {
 export const testNowMonitor = async (configId: string): Promise<TestNowResponse | undefined> => {
   return await apiService.get(API_URLS.TRIGGER_MONITOR + `/${configId}`);
 };
+
+export const fetchServiceEnabled = async (): Promise<{ serviceEnabled: boolean }> => {
+  return await apiService.get(API_URLS.SERVICE_ENABLED);
```

Comment:
Pretty sure I've also named my api variables and fetch functions similarly in my enablement PR. Let's sync to prevent as much naming confusing as possible.

Reply:
i have renamed everything from enabled to allowed.